packages/utils/blueprint-cli/src/publish/codecatalyst-authentication.ts

```diff
@@ -25,55 +26,70 @@ export function generateHeaders(authentication: CodeCatalystAuthentication, iden
   }
 }
 
-export const codecatalystAuthentication = async (log: pino.BaseLogger, options: {
-  cookie?: string;
-  endpoint: string;
-  region?: string;
-}): Promise<CodeCatalystAuthentication | undefined> => {
-
+export const codecatalystAuthentication = async (
+  log: pino.BaseLogger,
+  options: {
+    cookie?: string;
+    endpoint: string;
+    region?: string;
+  },
+): Promise<CodeCatalystAuthentication | undefined> => {
   if (options.cookie) {
     return {
       type: 'cookie',
       token: options.cookie,
     };
-  }
-  log.warn('Attempting to fetch credentials from AWS CLI profile.');
-  try {
-    const codecatalyst = new CodeCatalystClient({
-      region: options.region,
-      endpoint: path.join('https://', options.endpoint),
-    });
+  } else if (process.env.AWS_CONTAINER_TOKEN_ENDPOINT) {
+    log.warn('Attempting to fetch credentials from the CodeCatalyst workflow action environment.');
+    try {
+      const response = await axios.default.get(process.env.AWS_CONTAINER_TOKEN_ENDPOINT);
+      const credentials = response.data as { AccessKeyId: string; SecretAccessKey: string };
+      return {
+        type: 'workflowtoken',
+        token: `${credentials.AccessKeyId}:${credentials.SecretAccessKey}`,
```

Comment:
I believe this `token` gets passed directly as the `authorization` header so we should prepend `Bearer` to it.

Reply:
Nice catch!